sky/callbacks/sky_callback/integrations/keras.py

```diff
@@ -1,11 +1,10 @@
 """SkyCallback integration with Keras."""
 from typing import Dict, Optional
 
-import tensorflow as tf
-from tensorflow import keras
-
 from sky_callback import base
 from sky_callback import utils
+from tensorflow import keras
+import tensorflow as tf
```

Comment:
```suggestion
from tensorflow import keras
import tensorflow as tf

# isort: split
from sky_callback import base
from sky_callback import utils
```
We may want to do this since within this dir, "sky_callback" is considered first-party.